hooks/useAcademics.ts

```diff
@@ -0,0 +1,9 @@
+import { useAPI } from '../contexts/useAPI';
+
+export const useAllAcademicCategories = () => {
+  return useAPI().useQuery(['allAcademicCategoriesGet', null], {});
```

Comment:
the options param is optional so we can just omit it if were not using it

```suggestion
  return useAPI().useQuery(['allAcademicCategoriesGet', null]);
```